src/test/java/uk/gov/hmcts/darts/courtlogs/GetCourtLogRouteTest.java

```diff
@@ -21,7 +27,7 @@ class GetCourtLogRouteTest {
     private final OffsetDateTime offsetEndTime = OffsetDateTime.now();
 
     @Mock
-    private DartsFeignClient dartsFeignClient;
+    private CourtLogsClient dartsFeignClient;
```

Comment:
Not a big thing, but now we have separated the feign clients per resource, should name the field specifically?

Reply:
yep sorry my bad. Again really good spot. I did have this on my mind at one point or another. Thanks